docs/API.md

```diff
@@ -285,7 +295,7 @@ A representation of a folder in the source system
 
 | Name | Type | Description |
 | --- | --- | --- |
-| assets | <code>Array.&lt;Asset&gt;</code> | the retrieved assets |
+| assets | [<code>Array.&lt;AssetData&gt;</code>](#AssetData) | the retrieved assets |
```

Comment:
I am confused why the franklin api doc md templates use <code> instead of backticks

Reply:
ugh the autoformatting did not show that comment correctly lol
![Screen Shot 2023-01-26 at 8 06 11 AM](https://user-images.githubusercontent.com/20048485/214886593-8b228e8a-67a6-4567-b330-2d34eb2e3b32.png)